setup.py

```diff
@@ -93,7 +93,7 @@ def read(*rnames):
       extras_require={
           'test': test_requires,
       },
-      test_requires=test_requires,
+      tests_require=test_requires,
```

Comment:
That was pyroma:
```
$ pyroma -a .
------------------------------
Checking .
//distutils/dist.py:267: UserWarning: Unknown distribution option: 'test_requires'
  warnings.warn(msg)
```

It bugs me that `extras_requires`, `tests_require`, `install_requires` and `setup_requires` are inconsistent with each other, subject-verb agreement or not.